src/reuse/modules/mobile/userInteraction.ts

```diff
@@ -61,7 +63,126 @@ export class UserInteraction {
         vl.log("Checkbox already selected.");
       }
     } catch (error) {
-      this.ErrorHandler.logException(error);
+      this.ErrorHandler.logException(error, "Error: During check", true);
+    }
+  }
+
+  /**
+   * @function doubleTap
+   * @memberof mobile.userInteraction
+   * @description Double Tap's on the mobile element.
+   * @param {Element | string} element - The element or CSS selector describing the element (e.g., accessibility ID, XPath, CSS,).
+   * @param {number} [timeout = 30000] - The timeout to wait(ms) (default: 30000).
+   * @example
+   * const elem = await mobile.userInteraction.doubleTap(elem);
+   * const elem = await mobile.userInteraction.doubleTap(elem, 20000);
+   */
+  async doubleTap(element: Element, timeout: number = parseFloat(process.env.QMATE_CUSTOM_TIMEOUT!) || 30000) {
+    const vl = this.vlf.initLog(this.doubleTap);
+
+    try {
+      vl.log("Expecting element to be displayed and enabled");
+      await Promise.all([
+        expect(element).toBeDisplayed({
```

Comment:
lets add a reuse function for the check -> `waitToBeEnabled` here: https://github.com/SAP/wdio-qmate-service/blob/main/src/reuse/modules/mobile/element.ts#L90

Reply:
its updated with previously added reusable method.